### PR TITLE
🐛(dockerfiles) fix copy after openedx download

### LINKED ISSUE #328: Failed to solve: failed to compute cache key: failed to calculate checksum of ref
## Expected Behavior

Can not run completed bootstrap command.

## Actual Behavior

We got issues with the download library.

## Steps to Reproduce

1. make bootstrap

## Specifications

 > [edxapp 5/8] COPY --from=downloads /downloads/edx-platform-* .:
------
failed to solve: failed to compute cache key: failed to calculate checksum of ref zp2wh8029gbqt1357kbr6hdhz::i1p0qru1gizriupvnxy4bzah1: "/downloads/edx-platform-open-release-ironwood.2/node_modules/edx-pattern-library/pattern-library/fonts": not found
make: *** [build] Error 17

Screenshot 2023-06-16 at 3.13.04 PM

### PR DESCRIPTION
**Pull request by kernicPanel**
## Purpose

After openedx download and extraction, the copy command was broken.

Fixes #328

Also, unmaintained release versions builds are useless, they can be removed.

## Proposal

Use EDX_RELEASE_REF to target the file to copy
Remove unmaintained release versions

### CODE REVIEW

**Review by jbpenrath — COMMENTED** [review]
Unfortunately, CI confirms our assumption. `EDX_RELEASE_REF` cannot be used to complete the path.

File: releases/dogwood/3/fun/Dockerfile [hunk 1]
```diff
@@ -90,7 +90,10 @@ RUN apt-get update && \
 
 WORKDIR /edx/app/edxapp/edx-platform
 
-COPY --from=downloads /downloads/edx-platform-* .
+# Get default EDX_RELEASE_REF value (defined on top)
+ARG EDX_RELEASE_REF
+COPY --from=downloads "/downloads/edx-platform-${EDX_RELEASE_REF}" .
```
Comment (sampaccoud) [inline releases/dogwood/3/fun/Dockerfile:95]: I don't understand what it fixes. Isn't copying the same files as before?

Comment (sampaccoud) [reply releases/dogwood/3/fun/Dockerfile:95]: Could the problem be that docker new versions don't support the * syntax? If yes, that would explain the recent appearance of this bug and maybe you can mention it in your commit message.

**Review by jmaupetit — COMMENTED** [review]
Looks good to me. I'll let @sampaccoud approve this PR once you answered its comment.

File: releases/dogwood/3/fun/Dockerfile (hunk 1 above)
Comment (kernicPanel) [reply releases/dogwood/3/fun/Dockerfile:95]: With `*` we get this error:
``` => ERROR [edxapp 5/8] COPY --from=downloads /downloads/edx-platform-* .                                                                                                                                                              0.0s
------
 > [edxapp 5/8] COPY --from=downloads /downloads/edx-platform-* .:
------
Dockerfile:95
--------------------
  93 |     # Get default EDX_RELEASE_REF value (defined on top)
  94 |     ARG EDX_RELEASE_REF
  95 | >>> COPY --from=downloads "/downloads/edx-platform-*" .
  96 |     
  97 |     
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref GMTM:JCH3:CSUP:6DLT:2WEI:K6OQ:VVP7:LZEN:ZH7C:PNGT:6354:O2QN::se85uz01n5chhjb7sldrl36w0: "/edx/app/edxapp/fun-config/django-settings/fun": not found
ERROR: Service 'lms' failed to build : Build failed
```

But docker documentation doesn't mention any problem with `*`: https://docs.docker.com/engine/reference/builder/#copy
> To add all files starting with "hom":
> COPY hom* /mydir/

So I don't understand where the bug comes from…

Comment (sampaccoud) [reply releases/dogwood/3/fun/Dockerfile:95]: from what I remember there is a symbolic link story in `/edx/app/edxapp/fun-config/django-settings/fun`?

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**